chromadb/__init__.py

```diff
@@ -12,10 +12,12 @@
 
 def configure(**kwargs):
     """Override Chroma's default settings, environment variables or .env files"""
+    global __settings
```

Comment:
can we remove this `global` and the other one? we are mainly concerned about unintended side effects.